tests/test_hydrotel.py

```diff
@@ -399,13 +398,13 @@ def test_run(self, tmpdir, test):
     def test_copypaste(self, tmpdir):
         xhydro.testing.utils.fake_hydrotel_project(tmpdir)
         # Remove simulation.csv
-        os.remove(tmpdir / "simulation" / "simulation" / "simulation.csv")
+        Path(tmpdir / "simulation" / "simulation" / "simulation.csv").unlink()
         Hydrotel(
             tmpdir,
             "SLNO.csv",
             use_defaults=True,
         )
-        assert os.path.exists(tmpdir / "simulation" / "simulation" / "simulation.csv")
+        assert Path(tmpdir / "simulation" / "simulation" / "simulation.csv").unlink()
```

Comment:
Does that work? It looks like `unlink` is used for two different things.

Reply:
That is very clearly typo. Nice catch!